chef_master/source/install_chef_automate.rst

```diff
@@ -243,7 +243,7 @@ Chef Automate can use push jobs to coordinate build jobs across build nodes when
 
 Push jobs is available as an add-on to Chef server. You can also use runners and the new job dispatch system instead of the previous push jobs-based system.
 
-.. note:: Chef Automate requires Push Jobs Server 1.x and is not compatible with Push Jobs Server 2.x.  If you are installing Chef Automate on Red Hat Enterprise Linux/CentOS 7, use the Red Hat Enterprise Linux/CentOS 6 package for Push Jobs Server 1.x (available at `<https://downloads.chef.io/push-jobs-server/redhat/>`_) and manually install it.  For other platforms, you can use the automated installation method for Push Jobs Server 1.x as described below.
+.. note:: Chef Automate requires Push Jobs Server 1.x and is not compatible with Push Jobs Server 2.x.  If you are installing Chef Automate on Red Hat Enterprise Linux/CentOS 7, use the Red Hat Enterprise Linux/CentOS 6 package for Push Jobs Server 1.x (available at `<https://downloads.chef.io/push-jobs-server/stable/1.16`_) and manually install it.  For other platforms, you can use the automated installation method for Push Jobs Server 1.x as described below.
```

Comment:
Download link should be https://downloads.chef.io/push-jobs-server/stable/1.1.6 (not 1.16) and you're missing a closing bracket as well.